### DISCUSSION TITLE
Shared or Synced Page Tree for Multisite

### DISCUSSION THREAD

**Discussion by sewid** [statamic/cms#6988] [Q&A] [answered]
Hi there,

I have a multisite setup for DE and EN locales. The main page tree is in DE and every page should have a EN translation. Is it possible to share a page tree for both translations, so that I can move a page in the german translation and the english version is moved as well?

Thanks!
Sebastian

**Comment by jasonvarga** [comment, marked as answer]
Not at the moment. The trees are separate per site. Feel free to open a feature request on the ideas repo.

**Reply by sewid** [reply]
Thanks, I just did that: https://github.com/statamic/ideas/issues/905
